src/tests/ftest/util/daos_core_base.py

```diff
@@ -49,6 +49,8 @@ def __init__(self, *args, **kwargs):
 
         test_timeout = self.params.get("test_timeout", self.TEST_PATH)
         if test_timeout:
+            self.log.info("Test timeout updated to {timeout}".format(
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments